doc/api/number/number-formatter.md

```diff
@@ -84,7 +84,7 @@ zhFormatter( 3.141592 );
 // > "三.一四二"
 ```
 
-Controlling digits by specifying integer and fraction digits counts:
+The number of decimal places can be decreased or increased using `minimumFractionDigits` and `maximumFractionDigits`.
```

Comment:
Maybe add headers to each example?
```md
#### Static Formatter
#### Instance Formatter
#### Customzing fractions
#### Customizing significant decimal digits
#### Formatting percentages
#### Controlling rounding
#### Reusing formatter instance
```
Not sure about "Customzing" and "Controlling", is there something better?